modules/asset-cdn/asset-cdn.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Plugin Name: Asset CDN
+ * Description: Speed up Javascript and CSS
+ * Plugin URI: https://github.com/automattic/jetpack
+ * Author: Automattic
+ * Author URI: https://automattic.com
+ * Version: 0.1.0
+ * Text Domain: asset-cdn
+ * Domain Path: /languages/
+ * License: GPLv2 or later
+ */
+
+/**
+ * TODO
+ * - versioning (combine ver hashes) and cachebusting
+ * - concat/minify/serve JS too
+ * - asset inlining for smaller styles?
+ * - critical CSS support?
+ * - non-enqueued assets?
+ */
+
+class Asset_CDN {
```

Comment:
We should probably prefix this with `Jetpack_`

Reply:
👍